src/libraries/System.Private.CoreLib/src/Internal/Console.iOS.cs

```diff
@@ -0,0 +1,29 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Text;
+
+namespace Internal
+{
+    public static partial class Console
+    {
+        public static unsafe void Write(string s)
+        {
+            fixed (char* ptr = s)
+            {
+                Interop.Sys.Log((byte*)ptr, s.Length * 2);
+            }
+        }
+        public static partial class Error
+        {
+            public static unsafe void Write(string s)
+            {
+                fixed (char* ptr = s)
+                {
+                    Interop.Sys.Log((byte*)ptr, s.Length * 2);
```

Comment:
Isn't there a `LogError` for iOS? Because before this was called, right?
https://github.com/dotnet/runtime/blob/bfa82aff0d168bf4dae2847b07f140e4c68c81eb/src/libraries/System.Private.CoreLib/src/Internal/Console.Unix.cs#L27

Reply:
good catch, yes this should be `Interop.Sys.LogError`

Reply:
Yes, it was accidentally replaced by last commit